web/src/routing/routes/ForgotPassword/ForgotPasswordView/ForgotPasswordForm/ForgotPasswordForm.tsx

```diff
@@ -0,0 +1,68 @@
+import { VFC } from "react";
+import { Controller, useForm } from "react-hook-form";
+import { EmailField } from "@components/EmailField/EmailField";
+import { Box, Button, CircularProgress, Typography } from "@mui/material";
+import { useStyles } from "./styles";
+import { ForgotPasswordFormState } from "./formState";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { forgotPasswordFormValidationSchema } from "./formValidationSchema";
+
+interface Props {
+  onSubmit(formData: ForgotPasswordFormState): void;
+  loading: boolean;
+  error?: string;
+}
+
+const defaultValues = {
+  email: "",
+};
+
+export const ForgotPasswordForm: VFC<Props> = ({
+  onSubmit,
+  error,
+  loading,
+}) => {
+  const classes = useStyles();
+
+  const {
+    handleSubmit,
+    control,
+    formState: { errors },
+  } = useForm<ForgotPasswordFormState>({
+    defaultValues,
+    resolver: yupResolver(forgotPasswordFormValidationSchema()),
+  });
+  return (
+    <form onSubmit={handleSubmit(onSubmit)}>
+      <Controller
+        name="email"
+        control={control}
+        render={({ field }) => (
+          <EmailField
+            onChange={field.onChange}
+            errorMessage={errors.email?.message}
```

Comment:
you are able to retrieve an error from `field` or some other argument in this method afair